packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -124,6 +124,7 @@ describe("Hotkeys", () => {
             // wait for the dialog to animate in
             setTimeout(() => {
                 expect(document.querySelector(".pt-hotkey-column")).to.exist;
+                expect(document.querySelector(".pt-overlay-open").classList.contains("pt-overlay-inline")).to.be.false;
```

Comment:
Yeah makes sense. Just figured a shoddy palisade would be better than no defense at all.